content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -27,59 +27,59 @@ Prisma's integrated seeding functionality expects the following:
 1. A `seed.ts` or `seed.js` file in the [same location as your `schema.prisma` file](../../concepts/components/prisma-schema#prisma-schema-file-location).
 2. **One** of the following inside the `seed.js`/`seed.ts` file:
 
-    * A script inside `seed.ts` / `seed.js`
```

Comment:
this one is a big mess

Reply:
List inside a list with a component inside that has code blocks - not surprising this 💥 
But maybe it actually renders correctly?

Reply:
@janpio it's not just crazy, it's next level crazy

https://user-images.githubusercontent.com/4734297/124755995-70044e80-df45-11eb-9974-d9c21b96877d.mp4

Reply:
Haha I think that is just prettier giving up completely and throwing up its hands: "If you want to mix Markdown lists, with a html like component and then put more Markdown code blocks in there, I'll show you what I think of that..." (I think the actual problem is the `<TabbedContent ...>` and `<tab>` that it can barely handle in Markdown)

Reply:
this one produced invalid mdx ([ast](https://astexplorer.net/#/gist/2befce6edce1475eb4bbec001356b222/35997d3b44347daabad8dd1a107adc22dd873de2))

Looks like 4+ space intention makes parser think that we're parsing a code block https://www.markdownguide.org/basic-syntax/#code-blocks

Reply:
I've annotated with `prettier-ignore` for now.